sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -248,11 +266,27 @@ def _validate(  # pylint: disable=no-self-use
         if isinstance(component, types.FunctionType):
             component = _refine_component(component)
 
-        # local validation only for now
-        # TODO: use remote call to validate the entire component after MFE API is ready
+        # local validation
         result = component._validate(raise_error=raise_on_failure)
+        # remote validation, note that preflight_operation is not available for registry client
```

Comment:
Does this mean for registry client, there won't be a remote validation even if specified `skip_remote_validation` to false? Shall we put this in doc string?

Reply:
Yes. There is a strange limitation that the validation must be based on a specific workspace. I'm discussing this with server-side. Will update this to docstring.